<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Server &amp;host:</string>
       </property>
       <property name="buddy">
        <cstring>serverHostLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serverHostLineEdit">
       <property name="text">
        <string>localhost</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Server &amp;port:</string>
       </property>
       <property name="buddy">
        <cstring>serverPortSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="serverPortSpinBox">
       <property name="minimum">
        <number>49152</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>56665</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toggleConnectionPushButton">
       <property name="text">
        <string>&amp;Connect</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="messageGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Message</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Client &amp;ID:</string>
          </property>
          <property name="buddy">
           <cstring>clientIdSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="clientIdSpinBox">
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>12345</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Client h&amp;ost:</string>
          </property>
          <property name="buddy">
           <cstring>clientHostLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="clientHostLineEdit">
          <property name="text">
           <string>localhost</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Client po&amp;rt:</string>
          </property>
          <property name="buddy">
           <cstring>clientPortSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="clientPortSpinBox">
          <property name="minimum">
           <number>49152</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>56789</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Message &amp;body:</string>
          </property>
          <property name="buddy">
           <cstring>messageBodyTextEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="messageBodyTextEdit">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Obey!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="sendPushButton">
        <property name="text">
         <string>&amp;Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
